src/wasm/types.rs

```diff
@@ -119,6 +120,13 @@ type CardanoCertificate =
         keypath: Keypath
         poolKeyhash: Uint8Array
       }
+    }
+  | {
+      voteDelegation: {
+        keypath: Keypath
+        type: CardanoDrepType
+        drepCredHash?: Uint8Array
```

Comment:
you changed the name here but the deserialization still uses `drepCredhash`, so if you tried to use `drepCredHash`, it would not actually show the hash.